gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -148,18 +155,35 @@ def ancestry_pca_eigenvalues(
 
 def get_relatedness_annotated_ht() -> hl.Table:
     """
-    Get the relatedness table annotated with get_relationship_expr.
+    Get the relatedness table annotated with relationship classifications for closely
+    related samples. We're not using `get_relationship_expr`, as it expects IBD values.
+    For v4, we only have IBS values from cuKING.
 
     :return: Annotated relatedness table
     """
-    relatedness_ht = relatedness.ht()
-    return relatedness_ht.annotate(
-        relationship=get_relationship_expr(
-            kin_expr=relatedness_ht.kin,
-            ibd0_expr=relatedness_ht.ibd0,
-            ibd1_expr=relatedness_ht.ibd1,
-            ibd2_expr=relatedness_ht.ibd2,
+    ht = relatedness.ht()
+    ibs0_2 = ht.ibs0 / ht.ibs2
+
+    # The classification thresholds were chosen by inspecting clusters formed in a
+    # scatter plot with KING kinship vs (IBD0 / IBD2) axes.
+    return ht.annotate(
+        relationship=hl.case()
+        .when(ht.kin < 0.1, UNRELATED)
+        .when(ht.kin > 0.42, DUPLICATE_OR_TWINS)
+        .when(
+            ht.kin < 0.34,
+            hl.case()
+            .when(
+                (ibs0_2 > 2e-4)
+                & ((ht.kin < 0.17) | (ibs0_2 < -1e-2 * ht.kin + 2.2e-3)),
+                SECOND_DEGREE_RELATIVES,
+            )
+            .when(ht.kin < 0.15, AMBIGUOUS_RELATIONSHIP)
+            .when(ibs0_2 < -3e-3 * ht.kin + 9.2e-4, PARENT_CHILD)
+            .when(ibs0_2 > 4e-5, SIBLINGS)
+            .default(AMBIGUOUS_RELATIONSHIP),
         )
+        .default(AMBIGUOUS_RELATIONSHIP)
     )
```

Comment:
I think we want to add all of this to a function probably also in `gnomad_methods` and with these options as defaults

Reply:
I wasn't sure what the best way to change the `gnomad_methods` implementation is, given that IBS values will behave quite differently from the PC-Relate IBD estimates. Should this be a new method within `gnomad_methods`, e.g. `get_ibs_relationship_expr`, so we don't break any code that currently relies on `get_relationship_expr`? Or should we add some sort of versioning / mode to the existing `get_relationship_expr`?

Reply:
Yeah, I was thinking a `get_ibs_relationship_expr` in `gnomad_methods` would be reasonable

Reply:
In contrast to the previous `get_relationship_expr`, the current IBS parameters for v4 unfortunately don't generalize well -- it's basically just an empirically determined set of lines to form clusters. Do you still think we should move it to `gnomad_methods` nevertheless?